### PR TITLE
Lab fix

### PR DESCRIPTION
**Pull request by TC-DarkSlayer**
- Due to kag issue, for drugs to be consumed when used as recipe, needs to be server_Die() when quantity is 0, else -1.
- Druglab/reactor new upgrade features from TC CTF
- Plants can now grow properly when it's at one block (kag issue stuffs)

### CODE REVIEW

File: TFlippy_TerritoryControl_Core_Dev/Gamemode/Rules/Scripts/Survival_Interface.as [hunk 1]
```diff
@@ -728,12 +728,18 @@ string getRank(string &in username, SColor &out col, CPlayer@ p)
 		}
 		break;
 
+		case 926613433: // blackguy123
+		{
+			col = SColor(255, 56, 149, 244);
+			return "TC CTF Creator";
```
Comment (Vam-Jam) [inline TFlippy_TerritoryControl_Core_Dev/Gamemode/Rules/Scripts/Survival_Interface.as:734]: what does this have to do with tc?

**Review by Vam-Jam — CHANGES_REQUESTED** [review]

**Review by laikadoggirl — COMMENTED** [review]
Stop hiding title changes everywhere retard

**Comment by laikadoggirl** [comment]
I see that "the last tc dev" title
